### PR TITLE
end2end qa pipeline (#167)

### PR DESCRIPTION
**Pull request by rosequ**
- Added a `setup.py` file
- Reorganized the bridge file.

### CODE REVIEW

**Comment by rosequ** [comment]
@nims11, @gauravbaruah please do a CR

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,7 @@
+from setuptools import setup
+
+setup(name='castor',
+      version='1.0.0',
+      description='models for question answering',
+      packages=['sm_model'],
+      )
```
Comment (salman1993) [inline setup.py]: Remove the comma on line 6 and end the parentheses there.

Comment (rosequ) [inline setup.py]: @salman1993 this follows the standard format: https://docs.python.org/3/distutils/setupscript.html
Can we ship it, if there's no other comment?

Comment (salman1993) [inline setup.py]: http://stackoverflow.com/questions/6609956/python-function-argument-list-formatting
i got confused by this but it's not a big deal. the comma at the end makes no sense though. ship it.

**Pull request merged**